### DISCUSSION TITLE
posts/python/Python3-mro-class

### DISCUSSION THREAD

**Discussion by giscus** [dreamhunter2333/dreamhunter-blog#16] [General]
# posts/python/Python3-mro-class

你指尖跃动的电光，是我此生不变的信仰

https://dreamhunter2333.com/posts/python/Python3-mro-class.html

**Comment by dreamhunter2333** [comment]
> 迁移数据
> "createdAt": "2021-06-02T08:33:12.065Z",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36",

你针猛
